packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -73,6 +74,10 @@ The act of filtering will use the \`hidden\` attribute on the options to remove
 It is recommended that if the \`Select\` has 15 or fewer options that you use the \`none\` setting for the \`filter-mode\`.
 `;
 
+const placeholderDescription = `
+To display placeholder text within the \`Select\` you must provide an option that has the \`disabled\`, \`selected\` and \`hidden\` attributes set. This option will not be available in the dropdown, and its contents will be used as the placeholder text.
```

Comment:
There's a bit of usage guidance about placeholder text in the [select IxD spec](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/select/specs/IxD.md#field-button). Assuming we still agree with it, we should replicate it here.

**Original text**
>Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that prompts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.

**My editorialized suggestion, feel free to tweak**
```suggestion
To display placeholder text within the \`Select\` you must provide an option that has the \`disabled\`, \`selected\` and \`hidden\` attributes set. This option will not be available in the dropdown, and its contents will be used as the placeholder text.

Any `Select` without a default selected option should provide placeholder text. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Use sentence casing and don't include punctuation at the end of the prompt.
```